Makefile

```diff
@@ -29,26 +29,26 @@ clean:
 services: init install
 	$(MAKE) -C services/nginz
 
-.PHONY: integration
-integration: fast
-	# We run "i" instead of "integration" to avoid useless rebuilds
-	# (since after "fast" everything will be built already)
+.PHONY: i
+i:
 	$(MAKE) -C services/cargohold i
 	$(MAKE) -C services/galley i
 	$(MAKE) -C services/brig i
 	$(MAKE) -C services/gundeck i-fake-aws
 	$(MAKE) -C services/spar i
 
+integration: fast i
+
 .PHONY: integration-aws
```

Comment:
```suggestion
.PHONY: i-aws
```

Reply:
thanks!  (oh man, that shouldn't have been so hard...)